backend/config/urls.py

```diff
@@ -63,3 +63,6 @@
     path("swagger/", schema_view.with_ui("swagger", cache_timeout=0), name="schema-swagger-ui"),
     re_path("", TemplateView.as_view(template_name="index.html")),
 ]
+
+if settings.BASE_URL:
+    urlpatterns = [path(f'{settings.BASE_URL.strip("/")}/', include(urlpatterns))]
```

Comment:
These lines seem to make the URL (e.g. <http://127.0.0.1:8000/v1/projects>) inaccessible. `settings.BASE_URL` is evaluated as `True`.